src/d/actor/d_a_obj_barrel2.cpp

```diff
@@ -45,139 +47,34 @@ const dCcD_SrcCyl daObjBarrel2::Act_c::M_cyl_src = {
     },
     // cM3dGCylS
     {
-        /* Center */ 0.0f, 0.0f, 0.0f,
+        /* Center */ 0.0f,
+        0.0f,
+        0.0f,
         /* Radius */ 50.0f,
         /* Height */ 100.0f,
     },
 };
 const daObjBarrel2::Attr_c daObjBarrel2::Act_c::M_attr[] = {
     {
-        0x5,
-        0x9,
-        0x520,
-        200.0f,
-        300.0f,
-        800.0f,
-        100.0f,
-        50.0f,
-        0.0099999998f,
-        4.0f,
-        1.0f,
-        3.0f,
-        1.0f,
-        500.0f,
-        300.0f,
-        25.0f,
-        0.0f,
-        0.0f,
-        0.0f,
-        0.0f,
-        0.0f,
-        0.55f,
-        0.5f,
-        0x3E8,
-        0.3f,
-        0.02f,
-        0.039999999f,
-        0.34999999f,
-        1.0f,
-        1.0f,
+        0x5,  0x9,   0x520, 200.0f, 300.0f, 800.0f, 100.0f,       50.0f,       0.0099999998f, 4.0f,
+        1.0f, 3.0f,  1.0f,  500.0f, 300.0f, 25.0f,  0.0f,         0.0f,        0.0f,          0.0f,
+        0.0f, 0.55f, 0.5f,  0x3E8,  0.3f,   0.02f,  0.039999999f, 0.34999999f, 1.0f,          1.0f,
```

Comment:
Here too. It's better for these to be on separate lines so that field name comments can be put on the left. Autoformatters screw things up, so it's better to revert to before you ran the formatter.